### DISCUSSION TITLE
[INDEXER REQ] LearnBits

### DISCUSSION THREAD

**Discussion by bulletproof2k** [autobrr/autobrr#1685] [Ideas]
- Indexer name: LearnBits
- Short name: LBS
- URL: http://learnbits.me/

- Network name: LearnBits official IRC (Internet Relay Chat)
- Network address: [irc.learnbits.me](irc://irc.learnbits.me)
- Port: 6667 or port 6670 for SSL connections
- SSL/TLS: Yes
channels: #LB-announce

[Announce category - Site category name](http://learnbits.me/irc.php)

**Comment by zze0s** [comment]
Hey @bulletproof2k, could you please gather the announce examples and categories for us? Will make it much easier 🙂
